app/generator/content/part3.js

```diff
@@ -0,0 +1,7 @@
+const part3 = (_statement) => {
```

Comment:
do we need to pass statement in if we dnt use it?

Reply:
As above. It will on the next ticket.